pkg/apps/blackduck/v1/creater.go

```diff
@@ -165,14 +165,20 @@ func (hc *Creater) Ensure(blackduck *v1.Blackduck) error {
 	}
 
 	// TODO wait for webserver to be up before we register
-	if err := hc.registerIfNeeded(blackduck); err != nil {
-		return err
+	if len(blackduck.Spec.LicenseKey) > 0 {
+		if err := hc.registerIfNeeded(blackduck); err != nil {
+			log.Infof("couldn't register blackduck %s: %v", blackduck.Name, err)
+		}
 	}
 
 	if strings.ToUpper(blackduck.Spec.ExposeService) == "NODEPORT" {
-		newBlackuck.Status.IP, err = hc.getLoadBalancerIPAddress(blackduck.Spec.Namespace, "webserver-lb")
+		newBlackuck.Status.IP = ""
```

Comment:
Can we potentially reuse [`GetIPAddress`](https://github.com/blackducksoftware/synopsys-operator/blob/master/pkg/blackduck/util/helpers.go#L55) here?